src/parts/Sidebar/SidebarContent/index.tsx

```diff
@@ -32,8 +32,8 @@ const SidebarContent = React.forwardRef<Ref, Props>(
           'flex-1',
           'flex',
           'flex-col',
-          { 'bg-white': process.env.REACT_APP_RELAY_CHAIN_NAME === POLKADOT },
-          { 'dark:bg-kintsugiMidnight': process.env.REACT_APP_RELAY_CHAIN_NAME === KUSAMA }
+          { 'bg-interlayHaiti-50': process.env.REACT_APP_RELAY_CHAIN_NAME === POLKADOT },
```

Comment:
This has to be set even though the bg colour is the same as the main container background, as removing it causes an issue with the navigation overlay on small screens.